src/alert/alert.style.tsx

```diff
@@ -143,3 +151,35 @@ export const TextContainer = styled.div`
     flex: 1;
     overflow-wrap: anywhere;
 `;
+
+export const TextWrapperContainer = styled.div<TextWrapperContainerStyleProps>`
+    ${(props) => {
+        const gradient =
+            "linear-gradient(to bottom, black 50%, transparent 100%)";
+        if (!props.$showMore && props.$collapsedHeight)
+            return `
+                height: ${props.$collapsedHeight}px;
+				overflow: hidden;
+                -webkit-mask-image: ${gradient};
+                mask-image: ${gradient};
+			`;
+    }}
+`;
+
+export const ShowMoreButton = styled.div`
```

Comment:
semantically should be a button

and will need `type="button"` set to avoid unintended form submissions